app/models/attachment.rb

```diff
@@ -7,7 +8,11 @@ class Attachment
   embedded_in :object, polymorphic: true
 
   def as_json *args
-    result = attributes.symbolize_keys.merge asset: asset.as_json
+    result = attributes.symbolize_keys.except(:_id, :_type, :asset_id).merge(
```

Comment:
It is ok, but i'm going to change the back to `_id` to make it consistant with the rest of the API